test/e2e/specs/editor-tracking/editor-tracking__pattern-events.ts

```diff
@@ -73,26 +72,25 @@ describe( DataHelper.createSuiteTitle( 'Editor tracking: Pattern-related events'
 		describe( 'From the inline inserter', function () {
 			// We use a different pattern here for distinction.
 			// This especially helps distinguish the toast popups that confirm patter insertion.
-			const patternName = 'Pricing table'; // This returns one result.
-			const patternNameInEventProperty = 'pricing-table';
+			const patternName = 'About Me Card';
+			const patternNameInEventProperty = 'a8c/about-me-card';
 
 			it( 'Clear event stack for clean slate', async function () {
 				await eventManager.clearEvents();
 			} );
 
 			it( 'Add pattern from inline inserter', async function () {
-				const openInlineInserter: OpenInlineInserter = async ( editor ) => {
-					const editorGutenbergComponent = new EditorGutenbergComponent( page, editor );
```

Comment:
With the changes proposed in this PR, the `EditorGutenbergComponent` POM now has two parameters instead of one. 

Instead of updating the initialization of `EditorGutenbergComponent` in this line to accept two parameters, I felt that a minor refactor to the inline block inserter was preferable.

According to p1673990937312019-slack-CQD1HH4MA, the inner function and "raw" `EditorGutenbergComponent` is used because there are many inline inserters (at the editor canvas top-level and nested within blocks) and there needs to be a way to specify which inline inserter to use.

Instead, I chose to refactor the inline pattern inserter to accept the Locator of the inline inserter itself. The `addPatternInline` accepts a locator to the intended inline block inserter instead of a function, and the rest is the same.